src/components/Votes.jsx

```diff
@@ -0,0 +1,62 @@
+// import { useEffect, useState } from "react";
+// import { patchArticleVotes } from "../utils/api";
+
+// export default function Votes({ article_id, initialVotes, onVoteChange }) {
+//   const [votes, setVotes] = useState(initialVotes);
+
+//   const handleVote = (increment) => {
+//     patchArticleVotes(article_id, increment).then((updatedArticle) => {
+//       setVotes(updatedArticle.votes);
+//       onVoteChange(updatedArticle.votes);
+//     });
+//   };
+
+//   return (
+//     <div className="votes-container">
+//       <p>Votes: {votes}</p>
+//       <button onClick={() => handleVote(1)}>Upvote</button>
+//       <button onClick={() => handleVote(-1)}>Downvote</button>
+//     </div>
+//   );
+// }
+
+import { useState, useEffect } from "react";
+import { patchArticleVotes } from "../utils/api";
+
+export default function Votes({ article_id, initialVotes, onVoteChange }) {
+  const [votes, setVotes] = useState(initialVotes);
+
+  useEffect(() => {
+    setVotes(initialVotes);
```

Comment:
votes is already declared as initialVotes so why are you setting it again here ?